supervision/utils/video.py

```diff
@@ -192,6 +725,7 @@ def get_video_frames_generator(
     video.release()
 
 
+@DeprecationWarning
```

Comment:
let's use `deprecated` from `supervision.utils.internal`. it is currently used in the codebase to mark several deprecations.